ThreeBotPackages/threefold/tfgrid_directory/jobs/sync_directory.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from Jumpscale import j
+
+timeformat = "%a, %d %b %Y %H:%M:%S %Z"
+
+def sync_directory():
+    bcdb = j.data.bcdb.get("tf_directory")
+    farm_model = bcdb.model_get(url="tfgrid.farm.1")
+    node_model = bcdb.model_get(url="tfgrid.node.2")
+
+    response = j.clients.threefold_directory.client.ListFarmers()[1]
+    farms = response.json()
+
+    for farm in farms:
+        farmobjs = farm_model.find(name=farm["name"])
+        if not farmobjs:
+            farm_object = farm_model.new(farm).save()
+        else:
+            farm_object = farmobjs[0]
+
+        nodes = j.clients.threefold_directory.client.ListCapacity(query_params={"farmer": farm["iyo_organization"]})[1].json()
+        for node in nodes:
+            if node_model.find(node_id=node["node_id"]):
+                continue
```

Comment:
This is not correct. If you continue here. you will never update the value from the directory that could have change over time.